lua/glance/lsp.lua

```diff
@@ -19,16 +18,17 @@ local function create_handler(method)
           )
           return handler({})
         end
-        if results == nil or #results == 0 then
+        if result == nil or vim.tbl_isempty(result) then
```

Comment:
When LSP return a single Location, `#results` return zero, but in truth is one. See [https://stackoverflow.com/a/10755309](https://stackoverflow.com/a/10755309).

The Neovim core use [`vim.tbl_is_empty`](https://github.com/neovim/neovim/blob/29b80f6f2e9391b5d78390f65d09f00f73829310/runtime/lua/vim/lsp/handlers.lua#L360) to check if a table is empty.